lib/tasks/fhir.rake

```diff
@@ -0,0 +1,50 @@
+require "faker"
+
+namespace :fhir do
+  desc "Export simple patient-related data as fhir resources"
+  task :export, [:file_path] => :environment do |_task, args|
+    patients = Patient.order(:created_at)
+    patients = remove_pii(patients)
+    file_path = args[:file_path]
+    resources = []
+
+    patients.each do |patient|
+      resources << OneOff::Fhir::PatientExporter.new(patient).export
+      resources << patient.blood_pressures.map { |bp|
+        OneOff::Fhir::BloodPressureExporter.new(bp).export
+      }
+      resources << patient.blood_sugars.map { |bs|
+        OneOff::Fhir::BloodSugarExporter.new(bs).export
+      }
+      resources << patient.prescription_drugs.map { |drug|
+        OneOff::Fhir::PrescriptionDrugExporter.new(drug).export
+      }
+      resources << OneOff::Fhir::MedicalHistoryExporter.new(patient.medical_history).export
+      resources << patient.appointments.map { |appointment|
+        OneOff::Fhir::AppointmentExporter.new(appointment).export
+      }
+    end
+
+    File.open(file_path, "w+") do |f|
+      resources.flatten.each do |resource|
+        f.puts(resource.as_json.to_json)
```

Comment:
[q] is `as_json` needed? I'd have thought `to_json` would do the necessary conversion.

Reply:
We wanted to print each json on a single line. `to_json` adds newlines and indentation to prettify the json string